.editorconfig

```diff
@@ -222,11 +222,6 @@ dotnet_diagnostic.CA1303.severity = none                          # Literal stri
 # working around bug
 dotnet_diagnostic.RS0041.severity = none                          # Public members should not use oblivious types
 
-# TODO review these diagnostics as many/all are probably valid
-dotnet_diagnostic.CA1304.severity = none                          # Method behaviour depends upon user's locale
-dotnet_diagnostic.CA1305.severity = none                          # Method behaviour depends upon user's locale
-dotnet_diagnostic.CA1307.severity = none                          # Method behaviour depends upon user's locale
```

Comment:
Is the default "warning" here?

Reply:
I got warnings. The [docs](https://docs.microsoft.com/visualstudio/code-quality/ca1307) categorizes the rules as globalization warnings.